src/gui/Src/Gui/CPUDisassembly.cpp

```diff
@@ -102,8 +102,17 @@ void CPUDisassembly::mouseDoubleClickEvent(QMouseEvent* event)
 
     // (Disassembly) Assemble dialog
     case 2:
-        assembleSlot();
-        break;
+    {
+        duint followJumpsAndCalls = 0;
+        duint dest = DbgGetBranchDestination(rvaToVa(getInitialSelection()));
+
+        if(BridgeSettingGetUint("Gui", "FollowJumpsAndCalls", &followJumpsAndCalls) && followJumpsAndCalls && DbgMemIsValidReadPtr(dest))
+            gotoAddress(dest);
+
+        else
+            assembleSlot();

```

Comment:
For this I'd be happy to change the default behavior to `gotoAddress` and just get rid of `assembleSlot` altogether.